### ISSUE TITLE
Epochs for semantic segmentation finetuning on nuscenes mini and full?

### ISSUE THREAD

**Issue by barzanisar** [PRBonn/TARL#12]
Hey, 

How many epochs did you use for semantic segmentation finetuning on nuscenes mini and full? Also, did you use a learning rate scheduler? What was the optimizer and the learning rate? Can you please elaborate on the fine-tuning protocol you used for nuscenes?

Thanks!

**Comment by nuneslu** [comment]
Hi, the training protocol on nuscenes was the same as in SemanticKITTI dataset.
